src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.ServiceModel.Duplex.csproj">
-      <SupportedFramework>net46;netcore50;dnxcore50</SupportedFramework>
+      <SupportedFramework>net45;win8;dnxcore50</SupportedFramework>
```

Comment:
@ericstj
I see that in the S.SM.Duplex.nuproj "net45" and "win8" were listed as InboxOnTargetFramework, is that why they needed to be added to the SupportedFramework tag?

If net46 and netcore50 need to be removed why are they needed in the S.SM.Http.pkgproj? What is different about them?